charts/fluid/fluid/crds/data.fluid.io_databackups.yaml

```diff
@@ -1,13 +1,32 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why changing CRD version back from `v1` to `v1beta1`. I think this version CRD is deprecated in K8s 1.22.

Reply:
This is auto generated.